pages/new-hackers.md

```diff
@@ -8,3 +8,109 @@ permalink: /new-hackers/
 ### Civic Hacking 101
 
```

Comment:
It would be great to have a short description of civic hacking along with the video.